From 048a7cbca7a95799a6af0bcd85874c55b201118d Mon Sep 17 00:00:00 2001
From: maximilian attems <maks@stro.at>
Date: Mon, 22 Feb 2021 22:24:06 +0100
Subject: [PATCH 3/6] WHENCE: add missing symlink for BananaPi M3

Fixes (Debian bug #982579):
> [   11.957171] brcmfmac mmc2:0001:1: firmware: failed to load
brcm/brcmfmac43430-sdio.sinovoip,bpi-m3.txt (-2)
> [   11.967106] firmware_class: See https://wiki.debian.org/Firmware for
information about missing firmware
> [   11.977035] brcmfmac mmc2:0001:1: firmware: failed to load
brcm/brcmfmac43430-sdio.txt (-2)
> [   12.994756] brcmfmac: brcmf_sdio_htclk: HT Avail timeout (1000000): clkctl
0x50

Reported-by: Bernhard <bewoern1@gmail.com>
Signed-off-by: maximilian attems <maks@stro.at>
Signed-off-by: Josh Boyer <jwboyer@kernel.org>
---
 WHENCE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WHENCE b/WHENCE
index 11c0970..b569990 100644
--- a/WHENCE
+++ b/WHENCE
@@ -2717,6 +2717,7 @@ File: "brcm/brcmfmac43430-sdio.AP6212.txt"
 Link: brcm/brcmfmac43430-sdio.sinovoip,bpi-m2-plus.txt -> brcmfmac43430-sdio.AP6212.txt
 Link: brcm/brcmfmac43430-sdio.sinovoip,bpi-m2-zero.txt -> brcmfmac43430-sdio.AP6212.txt
 Link: brcm/brcmfmac43430-sdio.sinovoip,bpi-m2-ultra.txt -> brcmfmac43430-sdio.AP6212.txt
+Link: brcm/brcmfmac43430-sdio.sinovoip,bpi-m3.txt -> brcmfmac43430-sdio.AP6212.txt
 File: "brcm/brcmfmac43430-sdio.Hampoo-D2D3_Vi8A1.txt"
 File: "brcm/brcmfmac43430-sdio.MUR1DX.txt"
 File: "brcm/brcmfmac43430-sdio.raspberrypi,3-model-b.txt"
-- 
2.30.1

